pyproject.toml

```diff
@@ -1,9 +1,24 @@
 [tool.black]
+include = '\.pyi?$'
+exclude = '''
+(
+  /(
+      \.eggs
+    | \.git
+    | \.hg
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+
+)
+'''
 
 [build-system]
 requires = [
     "setuptools>=42",
     "wheel",
-    "pathlib",
```

Comment:
this was the issue, i must say didn't encoutnered it before. But `pathlib` is shipped with python so no need.